isis/src/qisis/apps/ipce/IpceMainWindow.cpp

```diff
@@ -211,31 +212,47 @@ namespace Isis {
 
     if ( qobject_cast<SensorInfoWidget *>(newWidget) ||
          qobject_cast<TargetInfoWidget *>(newWidget) ||
+         qobject_cast<ControlHealthMonitorView *>(newWidget) ||
          qobject_cast<TemplateEditorWidget *>(newWidget)) {
+      dock->setAllowedAreas(Qt::LeftDockWidgetArea | Qt::RightDockWidgetArea);
       splitDockWidget(m_projectDock, dock, Qt::Vertical);
+
+      // Save view docks for cleanup during a project close
+      m_specialDocks.append(dock);
     }
     else {
+      // The following logic not guaranteed to work as intented. If user moves one of the "special"
+      // views such as sensor from below the project dock to the right side of project, the following
+      // will put the new dock to the right of the moved "special" dock instead of tabbing.  The only
+      // way to possibly insure the intended functionality would be to check for the position of the
+      // last added dock and either add or tabify depending on location.  This might also allow the
+      // docks to be kept in a single list instead of m_specialDocks & m_viewDocks.
       if (m_viewDocks.count() == 0) {
-        splitDockWidget(m_projectDock, dock, Qt::Horizontal);
+        addDockWidget(Qt::RightDockWidgetArea, dock, Qt::Horizontal);
       }
       else {
         tabifyDockWidget(m_viewDocks.last(), dock);
         dock->show();
         dock->raise();
-        //  Keeps expanding IpceMainWindow to fit new dock
-        //  Below causes problems if the last dock has been tabbed, then a new view is created. All
-        // views except for first disappear.
-        //splitDockWidget(m_viewDocks.last(), dock, Qt::Horizontal);
       }
+
+      // Save view docks for cleanup during a project close
+      m_viewDocks.append(dock);
     }
 
     // When dock widget is destroyed, make sure the view it holds is also destroyed
     connect(dock, SIGNAL(destroyed(QObject *)), newWidget, SLOT(deleteLater()));
     // The list of dock widgets needs cleanup as each view is destroyed
     connect(dock, SIGNAL(destroyed(QObject *)), this, SLOT(cleanupViewDockList(QObject *)));
 
-    // Save view docks for cleanup during a project close
-    m_viewDocks.append(dock);
+    // Redmine #5471 TODO Hiding the centralWidget in this constructor gets rid of the space which
```

Comment:
1) spelling; 2) is this a constructor or still the ::addView method?; 3) should you say, maybe, testing hiding the central widget and then explain the behavior when you hide and when you don't?